htdocs/core/class/html.formticket.class.php

```diff
@@ -1608,13 +1612,6 @@ public function showMessageForm($width = '40%')
 			print '<input type="hidden" name="'.$key.'" value="'.$value.'">';
 		}
 
-		// Get message template
```

Comment:
Are you sure the getEMailTemplate is always already done ? Because line 1500 the call has the condition
`if (isset($this->param['models']) && $this->param['models'] != 'none') {`

Reply:
I think it was the case $this->param['models_id'] have to be set before from the caller htdocs/ticket/card.php
https://github.com/Dolibarr/dolibarr/blob/07c2feb69caf380ba1c07600fbf9f50011a4bbe5/htdocs/ticket/card.php#L1574
But may be, may be this method can be called from other pages from external module. 
So I re-add this getEMailTemplate with the same defaultfortype parameters to avoid to return email template with "All" type